bindings/py/packaging/src/htm/bindings/__init__.py

```diff
@@ -28,14 +28,13 @@ def _import_helper(name):
   from os.path import dirname, join
 
   # Fast path: see if the module has already been imported.
-  #print("name={}".format(name))
   try:
     return sys.modules[name]
   except KeyError:
     pass
 
   _mod = None
-  basename = name[15:]
+  basename = name[len("htm.bindings."):]
```

Comment:
Ah, this is something my shell script would not have found 👍

Reply:
CI found this one...